File: php/Latihaninput.php
<?php

$nama = readline("Masukkan Nama Barang: ");
$harga = floatval(readline("Masukkan Harga Barang: "));
$jumlah = intval(readline("Masukkan Jumlah Barang: "));

$total = $harga * $jumlah;

if ($total > 27900) {
    $diskon = $total * 0.09;
} else {
    $diskon = 0;
}

$bayar = $total - $diskon;

echo "===== STRUK PEMBELIAN =====\n";
echo "Nama Barang   : $nama\n";
echo "Harga Satuan  : $harga\n";
echo "Jumlah Beli   : $jumlah\n";
echo "Total Harga   : $total\n";
echo "Diskon        : $diskon\n";
echo "Total Bayar   : $bayar\n";
echo "===========================\n";

?>

File: c++/Latihaninput.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

main()
{
    string nama;
    float harga;
    int jumlah, diskon;

    cout << "Masukkan Nama Barang: ";
    cin >> nama;
    cout << "Masukkan Harga Barang: ";
    cin >> harga;
    cout << "Masukkan Jumlah Barang: ";
    cin >> jumlah;

    float total = harga * jumlah;

    if (total > 27900) {
        diskon = total * 0.09;
    } else {
        diskon = 0;
    }

    float bayar = total - diskon;


    cout << "=====STRUK PEMBELIAN=====" << endl;
    cout << "Nama Barang : " << nama << endl;
    cout << "Harga Satuan : " << harga << endl;
    cout << "Jumlah Beli : " << jumlah << endl;
    cout << "Total Harga : " << total << endl;
    cout << "Diskon : " << diskon << endl;
    cout << "Total Bayar : " << bayar << endl;
    cout << "=========================" << endl;
}